api/db/seeder/genreSeeder.ts

```diff
@@ -0,0 +1,30 @@
+import { Connection } from 'typeorm';
+import { Groups, Genre } from '../../entity';
+import { faker } from '@faker-js/faker';
+
+export type Location = 'Douai' | 'Lille';
+
+export async function genreSeeder(
+  connection: Connection,
+  amount = 10,
+): Promise<Groups[]> {
+  const res = [];
+  const repository = connection.getRepository(Genre);
+  const seenGenres = [];
+  for (let i = 0; i < amount; i++) {
+    const newGenre = faker.music.genre();
+    if (seenGenres.includes(newGenre)) continue;
+    else {
+      seenGenres.push(newGenre);
+      const genre = repository.create({
+        name: newGenre,
+      });
+      res.push(genre);
+      await genre.save();
+    }
+  }
```

Comment:
When you use continue , your can delete the `else` statement , see the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/continue) : 

```ts
  for (let i = 0; i < amount; i++) {
    const newGenre = faker.music.genre();
    if (seenGenres.includes(newGenre)) continue;
    seenGenres.push(newGenre);
    const genre = repository.create({
      name: newGenre,
    });
    res.push(genre);
    await genre.save();
  }
```

Otherwise maybe its more clear to just do something like this : 

```ts
  for (let i = 0; i < amount; i++) {
    const newGenre = faker.music.genre();
    if (!seenGenres.includes(newGenre)) {
      seenGenres.push(newGenre);
      const genre = repository.create({
        name: newGenre,
      });
      res.push(genre);
      await genre.save();
    }
  }
```